util/port_groups.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
```

Comment:
I understand this script was useful to do the migration. But once the file has been converted, the script is useless. Hence, shouldn't be part of the resulting codebase.